fio.run-file/fio.json

```diff
@@ -0,0 +1,135 @@
+{
+  "benchmarks": [
+    {
+      "name": "fio",
+      "ids": "1",
+      "mv-params": {
+        "global-options": [
+          {
+            "name": "required",
+            "params": [
+              {
+                "arg": "clocksource",
+                "vals": [
+                  "gettimeofday"
+                ]
+              },
+              {
+                "arg": "ramp_time",
+                "vals": [
+                  "5s"
+                ]
+              },
+              {
+                "arg": "unlink",
+                "vals": [
+                  "1"
+                ]
+              },
+              {
+                "arg": "filename",
+                "vals": [
+                  "/tmp/fio.foo"
+                ]
+              },
+              {
+                "arg": "runtime",
+                "vals": [
+                  "10s"
+                ]
+              },
+              {
+                "arg": "time_based",
+                "vals": [
+                  "1"
+                ]
+              },
+              {
+                "arg": "norandommap",
+                "vals": [
+                  "ON"
+                ]
+              },
+              {
+                "arg": "ioengine",
+                "vals": [
+                  "sync"
+                ]
+              },
+              {
+                "arg": "size",
+                "vals": [
+                  "10M"
+                ]
+              }
+            ]
+          }
+        ],
+        "sets": [
+          {
+            "include": "required",
+            "params": [
+              {
+                "arg": "bs",
+                "vals": [
+                  "64K",
+                  "128K",
+                  "256K"
+                ]
+              },
+              {
+                "arg": "rw",
+                "vals": [
+                  "read"
+                ]
+              }
+            ]
+          },
+          {
+            "include": "required",
+            "params": [
+              {
+                "arg": "bs",
+                "vals": [
+                  "4K",
+                  "8K"
+                ]
+              },
+              {
+                "arg": "rw",
+                "vals": [
+                  "randread"
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    }
+  ],
+  "tool-params": [
+    {
+      "tool": "sysstat"
+    },
+    {
+      "tool": "procstat"
+    }
+  ],
+  "tags": {
+    "run-type": "example"
+  },
+  "endpoints": [
+    {
+      "type": "remotehost",
+      "host": "foo.example.com",
+      "user": "root",
+      "client": "1",
+      "userenv": "rhubi8"
```

Comment:
all-in-one is all-in-one, users add their remotehost here and run crucible. If they want a helper script to replace vars, we could include a separate dir with these helper scripts.

Reply:
I'm trying to make this as simple as can be while also as obvious to the user as can be about what is being done.  I don't really want to come up with an elaborate library of functions to call that a user has to trace through to understand what is happening -- if possible.

Reply:
ack